api/libs/cloning/src/job-output.ts

```diff
@@ -5,5 +5,5 @@ export interface JobOutput {
   readonly componentId: string;
   readonly resourceId: string;
   readonly piece: ClonePiece;
-  readonly uri: string;
+  readonly uris: Array<{ uri: string; relativePath: string }>;
```

Comment:
could this be `ComponentLocation[]`?

Reply:
@hotzevzl the reason I didn't use tiny-type is the serialization/deserialization when the object hits the bull's redis - it wouldn't deserialize properly.